src/backend/stacks/AppointmentStack.ts

```diff
@@ -0,0 +1,15 @@
+import { StackContext, Api } from 'sst/constructs'
+
+export function AppointmentAPI({ stack }: StackContext) {
+  const api = new Api(stack, 'appointmentApi', {
+    routes: {
+      'GET /appointment/{id}': 'packages/functions/src/appointment.handler',
+      'GET /appointments': 'packages/functions/src/appointment.handler',
+      'POST /appointment': 'packages/functions/src/appointment.handler',
```

Comment:
in another project I did essentially
src/appointment/GET/handler.ts
src/appointment/{id}/GET/handler.ts
src/appointment/POST/handler.ts